pkg/kubelet/kubelet_test.go

```diff
@@ -716,53 +724,83 @@ func TestHandlePluginResources(t *testing.T) {
 
 	// pod requiring adjustedResource can be successfully allocated because updatePluginResourcesFunc
 	// adjusts node.allocatableResource for this resource to a sufficient value.
-	fittingPodSpec := v1.PodSpec{NodeName: string(kl.nodeName),
-		Containers: []v1.Container{{Resources: v1.ResourceRequirements{
-			Limits: v1.ResourceList{
-				adjustedResource: resourceQuantity2,
-			},
-			Requests: v1.ResourceList{
-				adjustedResource: resourceQuantity2,
+	fittingPodSpec := v1.PodSpec{
+		NodeName: string(kl.nodeName),
+		Containers: []v1.Container{
+			{
+				Resources: v1.ResourceRequirements{
+					Limits: v1.ResourceList{
+						adjustedResource: resourceQuantity2,
+					},
+					Requests: v1.ResourceList{
+						adjustedResource: resourceQuantity2,
+					},
+				},
+				ResourcesAllocated: v1.ResourceList{
+					adjustedResource: resourceQuantity2,
+				},
 			},
-		}}},
+		},
 	}
 	// pod requiring emptyResource (extended resources with 0 allocatable) will
 	// not pass PredicateAdmit.
-	emptyPodSpec := v1.PodSpec{NodeName: string(kl.nodeName),
-		Containers: []v1.Container{{Resources: v1.ResourceRequirements{
-			Limits: v1.ResourceList{
-				emptyResource: resourceQuantity2,
-			},
-			Requests: v1.ResourceList{
-				emptyResource: resourceQuantity2,
+	emptyPodSpec := v1.PodSpec{
+		NodeName: string(kl.nodeName),
+		Containers: []v1.Container{
+			{
+				Resources: v1.ResourceRequirements{
+					Limits: v1.ResourceList{
+						emptyResource: resourceQuantity2,
+					},
+					Requests: v1.ResourceList{
+						emptyResource: resourceQuantity2,
+					},
+				},
+				ResourcesAllocated: v1.ResourceList{
+					emptyResource: resourceQuantity2,
```

Comment:
rename resourceQuantity2 and resourceQuantity1 to more explanatory names. I have no idea what resourceQuantity2 is by looking at this part of the code

Reply:
I agree that those names are not intuitive. (in this case they seem to refer to zero/failed/adjusting resources. The upstream code has it this way - I guess unit tests have this pattern.  See: https://github.com/kubernetes/kubernetes/blob/release-1.15/pkg/kubelet/kubelet_test.go#L640

I'm just extending it for the new feature.